.gitignore

```diff
@@ -5,4 +5,5 @@ node_modules
 target/modules
 target/tests
 typings
+debug.log
```

Comment:
node generates it. I think I ran some invalid command. These were the contents of mine. 
```
[0227/160250:ERROR:tcp_listen_socket.cc(76)] Could not bind socket to 127.0.0.1:6004
[0227/160250:ERROR:node_debugger.cc(86)] Cannot start debugger server
```